config/locales/en.yml

```diff
@@ -873,19 +873,36 @@ en:
     articles_edited: Articles Edited
     articles_edited_generic: Pages Edited
     articles_edited_wikidata: Items Edited
+    article_scoped_program_info:
+        Your course is an 'Article Scoped Program', in which only assigned articles and categories
+        are counted.
     bytes_added: Bytes Added
     characters: Characters
     char_added: Chars Added
     char_changed: Characters Changed
     char: Chars Added
+    close_modal: Close
     date_time: Date/Time
     edit_count_description: Total Edits
+    error_count_message: We encountered %{error_count} errors while updating your course.
     last_reviewed: Last Reviewed
+    missing_data_heading: Missing or unexpected results?
+    outreach_help: Need Help? Refer to 'Report a problem' in the navigation bar.
     per_user: '%{number}/user'
     revisions: Recent Edits
     references_count: References Added
     references_doc: This is the number of reference tags added to articles, and can include multiple references to the same source. The data comes from the ORES article quality model, and is only available for some languages.
+    replag_info: >
+        One possibility of missing data might be replag. Sometimes our replica databases 
```

Comment:
The Dashboard retrieves edit data from WMF Cloud replica databases. Sometimes these databases are missing the most recent edits because of replication lag (replag). When this is the case, the missing edits will show up later on.

Reply:
Done!